nyumon3/index.html

```diff
@@ -0,0 +1,117 @@
+<!-- @format -->
+
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <!-- ウェブページの文字エンコーディングを指定するために使用されます。 -->
+    <meta charset="utf-8" />
+    <!-- ウェブページのビューポート（表示領域）の設定を指定します。 -->
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <!-- ウェブページのタイトルに使用されます。 -->
+    <title>Taro's Scenery</title>
+    <!-- ウェブページの簡潔な説明や概要を提供するために使用されます。 -->
+    <meta name="description" content="フォトグラファーの山田太郎です" />
+    <!-- モバイルブラウザが電話番号の検出やリンク化を行わないように設定しています。 -->
+    <meta name="format-detection" content="telephone=no" />
+    <!-- reset.cssは、destyleを使用しています。 -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/destyle.css@1.0.15/destyle.css"
+    />
+    <!-- スタイルシート（css）を呼び出しています -->
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300;400;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__inner wrapper">
+        <h1 class="header__title">
+          <img src="image/logo.png" />
+        </h1>
+        <nav class="header__nav">
+          <ul class="header__navList">
+            <li class="header__navItem"><a href="#service">Service</a></li>
+            <li class="header__navItem"><a href="#news">News</a></li>
+            <li class="header__navItem"><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+  
+    <div class="firstView">
+      <h2 class="firstView__title">My cooking</h2>
+    </div>
+  
+    <section id="service" class="service">
+      <div class="wrapper">
+        <h2 class="u-sectionTitle">Service</h2>
+        <p class="u-sectionSubTitle">サービス</p>
+        <ul class="service__cards">
+          <li class="service__card">
+            <img src="image/service.png" alt="service1" class="service__cardImage">
+            <div class="service__textbox">
+              <h3 class="service__cardTitle">h3見出し</h3>
+              <p class="service__cardDescription">テキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキ</p>
+            </div>
+          </li>
+          <li class="service__card">
+            <img src="image/service.png" alt="service2" class="service__cardImage">
+            <div class="service__textbox">
```

Comment:
```suggestion
            <div class="service__textBox">
```
こちらキャメルケースでお願いします🙇
全３箇所あり。